internal/restorer/restorer.go

```diff
@@ -406,19 +408,22 @@ func (res *Restorer) RestoreTo(ctx context.Context, dst string) error {
 					}
 				}
 				res.trackFile(location, updateMetadataOnly)
+				if !updateMetadataOnly {
+					restoredFileCount++
```

Comment:
@MichaelEischer I didn't see any thread/goroutines within RestoreTo.

Do you think I should make it concurrency safe?

Reply:
`traverseTree` is single threaded. Thus there's no need to be concurrency safe for now.